src/Workleap.DomainEventPropagation.Publishing/EventPropagationPublisherOptionsValidator.cs

```diff
@@ -11,6 +11,28 @@ public ValidateOptionsResult Validate(string name, EventPropagationPublisherOpti
             return ValidateOptionsResult.Fail("A token credential or an access key is required");
         }
 
+        if (string.IsNullOrWhiteSpace(options.TopicName))
+        {
+            return ValidateOptionsResult.Fail("A topic name is required");
+        }
+
+        if (options.TopicEndpoint is null or { Length: 0 })
```

Comment:
Is there an advantage to this instead of string.IsNullOrWhiteSpace?

Reply:
way better. I'm a little rusted obviously.

Reply:
no, I let autocomplete code for me.